vue/components/ui/molecules/input-slider/input-slider.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div class="input-slider" v-bind:class="classes">
+        <slider-ripe
+            v-bind:value.sync="valueData"
+            v-bind:disabled="disabled"
+            v-bind:min="min"
+            v-bind:max="max"
+            v-bind:step="step"
+            v-on:blur="onBlur"
+            v-on:focus="onFocus"
+        />
+        <input-symbol
+            v-bind:value="valueData"
+            v-bind:input-props="inputProps"
+            v-bind:disabled="disabled"
+            v-bind:width="100"
+            v-bind:height="32"
+            v-bind:symbol="symbol"
+            v-bind:variant="variant"
+            v-on:update:value="onUserInput"
+            v-on:blur="onBlur"
+            v-on:focus="onFocus"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.input-slider {
+    align-items: center;
+    display: flex;
+}
+
+.input-slider > .slider-ripe {
+    margin: 0px 10px 0px 0px;
+}
+</style>
+
+<script>
+export const InputSlider = {
+    props: {

```

Comment:
I'm thinking if we shouldn't just have `inputProps` and `sliderProps`. WDYT?

Reply:
I think it sounds good, doing a commit with the change